app/contact/contact.component.ts

```diff
@@ -1,13 +1,24 @@
-import { Component } from '@angular/core';
-import { MdCard } from '@angular2-material/card';
+import {Component, ViewEncapsulation} from '@angular/core';
```

Comment:
Please add whitespaces in the import statements:

`import { Component, ViewEncapsulation } from '@angular/core';`